build.gradle.kts

```diff
@@ -103,7 +103,7 @@ subprojects {
     apply(plugin = "java")
     apply(plugin = "org.kordamp.gradle.kotlin-project")
 
-    group = "io.saagie"
+    group = "io.saagie.updatarium"
```

Comment:
I don't understand this modification because it's already in master branch :
https://github.com/PierreLeresteux/updatarium/blob/master/build.gradle.kts#L110